src/main/java/puke/Ui.java

```diff
@@ -0,0 +1,98 @@
+package puke;
+
+import java.util.Scanner;
+
+/**
+ * Handles the user interface for the User interactions
+ */
+public class Ui {
+
+    /**
+     * Sends a corresponding system message based on given tasks and conditions
+     * @param i controller to decide which message to send
+     * @param d chatbot object
+     * @param t task that is being manipulated
+     */
+    public void systemMessage(int i , Duke d, Task t) {
+        if (i == 1) {
+            //Bye message
+            System.out.println("    ____________________________________________________________");
```

Comment:
Perhaps you can combine the strings into one single string and print once instead of printing multiple times. I noticed the same issue in several other places too.